src/components/CvGrid/CvColumn.vue

```diff
@@ -13,22 +13,27 @@ const props = defineProps({
   sm: {
```

Comment:
side note for this component story: it's not working
![image](https://github.com/carbon-design-system/carbon-components-vue/assets/20342514/bf91ec10-9360-40fd-95fc-4bb513ea803e)

Reply:
This one is **definitely** related to the lint changes. For the lint errors I added  `default: undefined` for the break-point size and this is causing the error. Updated to  `default: false`